src/editors/containers/ProblemEditor/components/EditProblemView/__snapshots__/index.test.jsx.snap

```diff
@@ -7,8 +7,9 @@ exports[`EditorProblemView component renders raw editor 1`] = `
   <div
     className="editProblemView d-flex flex-row flex-nowrap justify-content-end"
   >
-    <span
-      className="flex-grow-1"
+    <Container
```

Comment:
I had a quick look why the layout was behaving so off. Basically the Container is the wrong element for this, it gives some behavior that we don't want at all. Firstly the container (or some other class responsible) has margin auto, which centers the advanced editor (which we don't want). Secondly it also has a max-width which is also not good, because we want to have it grow organically to fill the empty space. So flex-grow-1 together with no margin and no max-width fixes the problem of the container being centered weirdly.